docs/amendment_and_voting.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Amendment and Voting
+id: amendment_and_voting
+author: Davis Sawali
+---
+
+In Tezos, the economic protocol can be amended by proposing and voting for changes. The protocol change will happen depending on the result of the votes.
+
+## Proposals
+A `Proposals` operation can be injected during a **Proposal Period**. It allows a delegate to submit a proposal identified by a protocol hash. The proposal with the most support is selected and will move on to the **Exploration Period**.
```

Comment:
Should we mention that "submit a proposal identified by a protocol hash" means "voting" for this protocol (and make it clear that ballot operation is not involved during the proposal period)?